src/main/java/core/basesyntax/service/impl/BalanceAction.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class BalanceAction implements ActionHandler {
+    @Override
+    public void apply(String fruit, int quantity) {
+        Storage.setBalance(fruit, quantity);
+    }
```

Comment:
I suggest you to move logic that you have in Storage.setBalance(... (and other Storage methods) to specific action, because in your case if logic will become more complicated you will require to add it to Storage class, but this class should be just storage holder, business logic should be performed by Action classes to align with Singe responsibility principle